### PR TITLE
Add recipe r-emlassemblyline

### PR DESCRIPTION
**Pull request by Marie59**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by dbast** [comment]
requires r-taxonomycleanr to be added.

**Comment by Marie59** [comment]
@dbast yes that's what I figured too so I did the recipe. It is ready here : 
https://github.com/conda-forge/staged-recipes/pull/20004#issuecomment-1295368655  
Could you merged it ? and also sorry to insist but I am waiting for adiv to be merged too (here : 
https://github.com/conda-forge/staged-recipes/pull/19007)
Could you help me ? 
Thank you for your help !

**Comment by Marie59** [comment]
@conda-forge/staged-recipes  and @conda-forge/help-r  the recipe is ready !

**Comment by jakirkham** [comment]
@conda-forge-admin, please ping team

File: recipes/r-emlassemblyline/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,96 @@
+{% set version = '3.5.4' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-emlassemblyline
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://github.com/EDIorg/EMLassemblyline/archive/refs/tags/v3.5.4.tar.gz
```
Comment (cbrueffer) [inline recipes/r-emlassemblyline/meta.yaml:10]:
```suggestion
  url: https://github.com/EDIorg/EMLassemblyline/archive/refs/tags/v{{ version }}.tar.gz
```

File: recipes/r-emlassemblyline/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,96 @@
+{% set version = '3.5.4' %}
```
Comment (cbrueffer) [inline recipes/r-emlassemblyline/meta.yaml:1]: 3.5.5 seems to be the latest version now.

**Comment by Marie59** [comment]
@cbrueffer and @conda-forge/staged-recipes the recipe is ready

**Pull request merged**